pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -233,13 +233,25 @@ func (r *ReconcileCephBlockPoolRadosNamespace) reconcile(request reconcile.Reque
 		return reconcile.Result{}, nil
 	}
 
+	// Build the NamespacedName to fetch the CephBlockPool and make sure it exists, if not we cannot
+	// create the rados namespace
+	cephBlockPool := &cephv1.CephBlockPool{}
+	pool := cephBlockPoolRadosNamespace.Spec.BlockPoolName
+	cephBlockPoolNamespacedName := types.NamespacedName{Name: pool, Namespace: request.Namespace}
+
 	if cephCluster.Spec.External.Enable {
-		logger.Debugf("external rados namespace %q creation is not supported, create it manually, the controller will assume it's there", namespacedName)
+		logger.Debug("skip creating external subvolume in external mode, create it manually, the controller will assume it's there")
```

Comment:
I don't understand this log message. It says we're skipping the creation, but then we are creating it in your new code block on line 250 below, right? (also there is a typo about subvolume instead of rados namespace)

Reply:
at line 250 we are not creating radosnamespace but we are creating clientProfile which csi operator resources.